doc/wiki/Contribute:-Update-Wiki.md

```diff
@@ -1 +1,87 @@
-TBF.
\ No newline at end of file
+We are glad you have something new to contribute to GAMER, and you would like to teach people how to use it!
+
+Before getting started, I recommend you have some basic knowledge of [repository, branch, fork](https://docs.github.com/en/repositories/creating-and-managing-repositories/about-repositories), [action](https://docs.github.com/en/actions), and [workflow](https://docs.github.com/en/actions/using-workflows).
+
+The followings are the outline of this document:
```

Comment:
How about adding one more section to explain that we have two separate places `doc` and `wiki` to store the information and what are their relations, in case people are confused on this page?

Reply:
Added.